treeherder/model/migrations/0009_remove_best_classification_and_is_verified_from_failure_line.py

```diff
@@ -12,12 +12,29 @@ class Migration(migrations.Migration):
     ]
 
     operations = [
-        migrations.RemoveField(
-            model_name='failureline',
-            name='best_classification',
-        ),
-        migrations.RemoveField(
```

Comment:
These need to be moved under `state_operations` so that Django knows the `RunSQL` is equivalent

Reply:
oh shoot, been a while since I've done SQL migrations! Thanks :)

Reply:
Updated and rebased into 862f1e5